src/test/ui/type-sizes.rs

```diff
@@ -102,6 +103,15 @@ enum Option2<A, B> {
     None
 }
 
+pub enum CanBeNicheFilledButShouldnt {
+    A(NonZeroU8, u32),
+    B
+}
+pub enum AlwaysTagged {
```

Comment:
```suggestion
pub enum AlwaysTaggedBecauseItHasNoNiche {
```

maybe make this self-documenting?